src/Cli/func/Azure.Functions.Cli.csproj

```diff
@@ -20,11 +20,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <BuildNumber Condition=" '$(BuildNumber)' == '' ">1</BuildNumber>
-    <MajorMinorProductVersion>4.0</MajorMinorProductVersion>
-    <Version>$(MajorMinorProductVersion).$(BuildNumber)</Version>
-    <AssemblyVersion>$(MajorMinorProductVersion).$(BuildNumber)</AssemblyVersion>
-    <FileVersion>$(MajorMinorProductVersion).$(BuildNumber)</FileVersion>
+    <MajorMinorProductVersion>4.0.10000</MajorMinorProductVersion>
```

Comment:
maybe too ambitious, but I want us to move away from this automatic build versioning that isnt making sense and hard to track